go.mod

```diff
@@ -45,6 +45,7 @@ require (
 	github.com/HdrHistogram/hdrhistogram-go v1.0.1 // indirect
 	github.com/beorn7/perks v1.0.1 // indirect
 	github.com/cespare/xxhash/v2 v2.2.0 // indirect
+	github.com/davecgh/go-spew v1.1.1 // indirect
```

Comment:
This is generated by `go mod tidy`.

Reply:
Checked locally and it does not get added:
```
yunikorn-core % go mod tidy
yunikorn-core % git status
On branch master
Your branch is up to date with 'origin/master'.
yunikorn-core % go version
go version go1.21.3 darwin/arm64
yunikorn-core % go mod why github.com/davecgh/go-spew
# github.com/davecgh/go-spew
(main module does not need package github.com/davecgh/go-spew)
```

Reply:
once switch to this PR's branch, `go mod tidy` shows the diff.